src/applications/vaos/components/NoValidVAFacilities.jsx

```diff
@@ -1,28 +1,88 @@
 import React from 'react';
-import { Link } from 'react-router';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import LocationDirectionsLink from 'applications/facility-locator/components/search-results/LocationDirectionsLink';
+import { titleCase } from '../utils/appointment';
+
+export default function NoValidVAFacilities({
+  systemId,
+  typeOfCare,
+  systemDetails,
+}) {
+  const attributes = systemDetails?.attributes;
+  const address = attributes?.address?.physical;
+  const phone = attributes?.phone;
+  const hours = attributes?.hours;
 
-export default function NoValidVAFacilities({ systemId }) {
   return (
     <div aria-atomic="true" aria-live="assertive">
       <AlertBox
         status="warning"
-        headline="Sorry, we couldn't find any locations for you"
+        headline={`We’re sorry. This location doesn’t allow online scheduling for ${typeOfCare} appointments`}
```

Comment:
Maybe we need to bring this up with Ryan and Peggy, but I wonder if saying "location" here is misleading. This message shows up when none of the locations in a VA system support this ToC. So it's really multiple locations that don't support this. And it might be confusing if a Veteran know that this specific facility isn't what they're trying to schedule into anyway and we're not giving him the facility choice he expects.

Reply:
@rtwell @peggygannon, I took the liberty of updating this alert title to be:

```We’re sorry. None of the facilities in this health system allow online scheduling for ${typeOfCare} appointments```

Please let me know if you have a better recommendation.

![image](https://user-images.githubusercontent.com/786704/70593044-8511d380-1b90-11ea-904a-66cb02c5e929.png)